packages/block-library/src/query-title/index.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import metadata from './block.json';
+import edit from './edit';
+import variations from './variations';
+
+const { name } = metadata;
+export { metadata, name };
+
+export const settings = {
+	title: __( 'Query Title' ),
```

Comment:
Should this one be "Query Title"? Or "Archive title"?
Also nitpick: Title should not be capitalized 😉

Reply:
>Title should not be capitalized 

Block titles seem to be capitalized, so I think it's okay for now.

Also Archive Title is the default block variation so it's what is shown in the inserter.